ChangeLog

```diff
@@ -42,6 +42,14 @@ Release date: TBA
 
   Closes #5281
 
+* Fix false positive - Allow unpacking of ``self`` in a subclass of ``typing.NamedTuple``.
```

Comment:
Just for the commit history I would be better to leave this where it currently is, right? This is not a blocking issue for me though.